deployment/helm/local-value/values.yaml

```diff
@@ -98,6 +98,7 @@ configs:
   networkRPCSysConfigName: BscTestNetworkRpc
   confirmBlocksCount: 0
   maxHandledBlocksCount: 5000
+  keptHistoryBlocksCount: 100000
```

Comment:
please add it into `deployment/helm/zkbas/values.yaml` as well.

Reply:
sure